src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -128,6 +128,12 @@ Copyright (c) .NET Foundation. All rights reserved.
                  FormatArguments="$(SdkResolverGlobalJsonPath)" />
   </Target>
 
+  <Target Name="_WarnWhenUsingNET6WithVS1611"
+        BeforeTargets="_CheckForInvalidConfigurationAndPlatform"
+        Condition="'$(BuildingInsideVisualStudio)' == 'true' and $(VisualStudioVersion.StartsWith('16'))">
```

Comment:
Shouldn't we be using the MSBuild `VersionLessThan` comparison function here?

Also, I think we should generate this error on the command line too, ie remove the check for `BuildingInsideVisualStudio`.  If you are using the MSBuild version from VS 16.11 from the command line, it is also not supported and you are likely to run into issues, so we should generate the warning there too.

Reply:
Sounds good, I'll remove the BuildingInsideVS check. As for the version comparison, that makes sense, but do we want to check if the version is exactly 16.11? Or check that it's anything below 17.0? (see this comment: https://github.com/dotnet/sdk/pull/20987#discussion_r708498680)

Reply:
We should check anything below 17.0, 16.10 or earlier will be at least as broken as 16.11.